cypress/locators/search/index.js

```diff
@@ -1,12 +1,19 @@
-import { SEARCH_COMPONENT } from './locators';
+import { SEARCH_COMPONENT, SEARCH_DEFAULT_ID, CROSS_ICON,
+  SEARCH_WITH_SEARCH_BUTTON_ID } from './locators';
+import { BUTTON_DATA_COMPONENT_PREVIEW } from '../button/locators';
+
+// DS locators
+export const searchDefault = () => cy.iFrame(SEARCH_DEFAULT_ID);
```

Comment:
Should we rename all those functions using `DS` in the end of each? To save naming convention?

Reply:
fixed